OneAppiOS/OneAppiOS/Assets.xcassets/updated_map.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "floorplan.png",
```

Comment:
IDT you should have this statically. Can this image be dynamically updated? We have http://hackru-misc.s3-website-us-west-2.amazonaws.com/floorplan.jpg for this. (Just a drive-by comment.)